pkg/acpi/raw.go

```diff
@@ -102,7 +99,7 @@ func (r *Raw) TableData() []byte {
 
 // Sig returns the table signature.
 func (r *Raw) Sig() string {
-	return fmt.Sprintf("%s", r.data[:4])
+	return string(r.data[:4])
```

Comment:
qq, are we sure at this point that data is long enough?

Reply:
No we don't, never been and this code can panic, but this transformation is save.